src/logic/encodable/basic.lean

```diff
@@ -333,6 +333,9 @@ nonempty.some $ let ⟨f, hf⟩ := exists_injective_nat α in ⟨of_inj f hf⟩
 
 end encodable
 
+lemma nonempty_encodable (α : Type*) [countable α] : nonempty (encodable α) :=
+⟨encodable.of_countable _⟩
```

Comment:
Why do you add this lemma? In case of `finite`/`fintype`, the `nonempty_fintype` lemma allows us to upgrade from `finite` to `fintype` without `classical.choice`. In this case, the upgrade needs `classical.choice` anyway.

Reply:
To be consistent. It also already existed for `finset` (`finset.nonempty_encodable`), so really I've only generalized it.